examples/taint-checking/taint_checking.bal

```diff
@@ -28,7 +28,7 @@ public function main(string... args) {
     // disallowing tainted data in the SQL query.
     //
     // This line results in a compiler error because the query is appended with a user-provided argument.
-    table dataTable = check customerDBEP->
+    table<string> dataTable = check customerDBEP->
```

Comment:
Pls change this to table<record {}>. We can't define tables with constraint type other than record types. It will give a runtime error in that case.